models/tiltrotor/tiltrotor.sdf.jinja

```diff
@@ -1035,9 +1038,6 @@
       <accelerometerBiasCorrelationTime>300.0</accelerometerBiasCorrelationTime>
       <accelerometerTurnOnBiasSigma>0.196</accelerometerTurnOnBiasSigma>
     </plugin>
-    <plugin name='groundtruth_plugin' filename='libgazebo_groundtruth_plugin.so'>
```

Comment:
So it doesn't publish groundtruth anymore?

Reply:
I added it back!